app/worker.py

```diff
@@ -21,15 +21,15 @@ def handler(event, context):
             raise Exception('Repository {}/{} does not exist'.format(config.GITHUB_ORG,
                                                                      config.GITHUB_REPOSITORY))
 
-        status_file = repository.file_contents(config.STATUS_FILE, ref='gh-pages')
+        status_file = repository.file_contents(config.STATUS_FILE, ref=config.BRANCH)
```

Comment:
Default value here is going to be ``master`` instead of ``gh-pages``. Is that expected?